.github/workflows/build-python-package.yml

```
@@ -25,15 +25,15 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Check JS existence
-        if: ${{ inputs.js == 'true' }}
+        if: ${{ inputs.js == true }}
```

The `== true` is unnecessary then isn't it?

```suggestion
        if: ${{ inputs.js }}
```

---

Looks like it. Just tested with `js: true` and `js: false` and both worked after removing the `== true`.

Kind of wish they just left everything as a string for consistency